yakut/param/formatter.py

```diff
@@ -66,21 +68,78 @@ def _make_json_formatter() -> Formatter:
     return lambda data: typing.cast(str, json.dumps(data, ensure_ascii=False, separators=(",", ":")))
 
 
+from collections.abc import Mapping, Collection, Sequence
```

Comment:
Imports should be located at the top of the file. PyLint is supposed to complain about this, too (check the CI logs).

Reply:
I will need to add pylint usage to the CONTRIBUTING.md